src/routes/GettingStartedPage/Wizard/GettingStartedWizard.tsx

```diff
@@ -0,0 +1,121 @@
+import React, { useState } from 'react';
+import { useWizardContext } from '@patternfly/react-core';
+import { Wizard, WizardFooter, WizardStep } from '@patternfly/react-core';
+
+import InitialSetup from './InitialSetup';
+import InstallOptions, {
+  Environment,
+  InstallationMethod,
+} from './InstallOptions';
+import Finish from './Finish';
+import InstallWithHelm from './InstallWithHelm';
+import InstallWithOperator from './InstallWithOperator';
+import AppLink from '../../../components/AppLink';
+
+const INITIAL_SETUP = 'Initial Setup';
+const OPTIONS = 'Options';
+const INSTALLATION = 'Installation';
+const FINISHING_UP = 'Finishing Up';
+
+// needed to append the select to on the InstallOptions page due to overlapping issues
+export const WIZARD_ID = 'getting-started-wizard';
+
+type CustomerFooterProps = {
+  selectedEnv: Environment | null;
+  selectedInstallation: InstallationMethod | null;
+};
+
+function CustomFooter({
+  selectedEnv,
+  selectedInstallation,
+}: CustomerFooterProps) {
+  const { activeStep, goToNextStep, goToPrevStep, close } = useWizardContext();
+
+  const isBackHidden = activeStep.name === INITIAL_SETUP;
+  const isNextDisabled =
+    activeStep.name === OPTIONS && (!selectedEnv || !selectedInstallation);
+
+  return (
+    <WizardFooter
+      activeStep={activeStep}
+      onNext={goToNextStep}
+      onBack={goToPrevStep}
+      onClose={close}
+      isNextDisabled={isNextDisabled}
+      isBackHidden={isBackHidden}
+      isCancelHidden
```

Comment:
is this intended to stay true? i don't see an analog in the old `js` file from what i can tell